test/tests/functional/pbs_nonprint_characters.py

```diff
@@ -127,28 +127,30 @@ def create_and_submit_job(self, user=None, attribs=None, content=None,
             use_attribs = {}
         else:
             use_attribs = attribs
-        retjob = Job(username=user, attrs=use_attribs)
+        retjob = Job(TEST_USER, attrs=use_attribs)
```

Comment:
why did you change `username=user` to `TEST_USER`?

Reply:
In some platforms job is submitted as root user, it causes problem while copying error & output files.
So I have changed it to TEST_USER

Reply:
then remove user argument from func